oio/cli/container/container.py

```diff
@@ -511,3 +513,57 @@ def take_action(self, parsed_args):
             account=account,
             container=parsed_args.container
         )
+
+
+class SnapshotContainer(lister.Lister):
+    """ Create a snapshot of a container.
+
+    Create a different database containing all the informations of the contents
+    in the database but with different chunks that are copy of the target
+    chunks at the time of the snapshot"""
```

Comment:
*Take a snapshot of a container*

*Create a separate database containing all information about the contents from the original database, but with copies of the chunks at the time of the snapshot.*